app/iht/views/application/tnrb/tnrb_overview.scala.html

```diff
@@ -70,6 +70,7 @@
                         @if(!CommonHelper.getOrException(CommonHelper.getOrException(registrationDetails.deceasedDetails).maritalStatus).equals(statusWidowed)) {
                             @tnrb_overview_table_row(id = "partner-marital-status",
                                 questionText = Html(Messages("iht.estateReport.tnrb.partner.married",
+                                    deceasedName,
                                     TnrbHelper.preDeceasedMaritalStatusSubLabel(widowCheckModel.dateOfPreDeceased),
                                     TnrbHelper.spouseOrCivilPartnerMessage(widowCheckModel.dateOfPreDeceased)
                                 )
```

Comment:
Lines 84-84, accessibility items, still use the phrase "the deceased" - this should be changed to the deceased's name in line with visible content. E.g. "page.iht.application.tnrbEligibilty.overview.partner.maritalStatus.question.screenReader.text.yes"